src/store/rule/actions.ts

```diff
@@ -15,14 +15,17 @@ const actions = {
       'SAVE_RULE_FAILURE',
   )<RequestPayload<RuleToSave>, CreateResponse<Rule | null>, CreateResponse<Rule | null>>(),
   setSavedRule: createAction('SET_CREATE_RULE')<CreateResponse<Rule | null>>(),
+  resetSavedRule: createAction('RESET_CREATE_RULE')<null>(),
   
   deleteRule: createAsyncAction(
       'DELETE_RULE_REQUEST',
       'DELETE_RULE_SUCCESS',
       'DELETE_RULE_FAILURE'
   )<RequestPayload<string>, DeleteResponse<string | null>, DeleteResponse<string | null>>(),
   setDeletedRule: createAction('SET_DELETED_RULE')<DeleteResponse<string | null>>(),
+  resetDeletedRule: createAction('RESET_DELETED_RULE')<null>(),
```

Comment:
Codacy has a fix for the issue: [Replace `'RESET_DELETED_RULE'` with `"RESET_DELETED_RULE"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  resetDeletedRule: createAction("RESET_DELETED_RULE")<null>(),
```